scripts/check-spec.sh

```diff
@@ -52,8 +52,14 @@ for package in $(git diff --name-only origin/master | grep ".rs$" | cut -d "/" -
     fi
 done
 
+check_spec_result=0
+
 header "Checking spec for Gear runtime"
 check_spec "$SPEC_ON_MASTER" "$ACTUAL_SPEC_GEAR" "$CHANGES"
 
 header "Checking spec for Vara runtime"
 check_spec "$SPEC_ON_MASTER" "$ACTUAL_SPEC_VARA" "$CHANGES"
+
+if [[ $check_spec_result -ne 0 ]]; then
+  exit 1
+fi
```

Comment:
Rename CHECK_SPEC_RESULT into EXIT_CODE and won't approach below be easier?
```suggestion
exit $EXIT_CODE
```